plugin.cpp

```diff
@@ -114,28 +114,46 @@ static const char *default_config = QUOTE({
 		"order" : "9",
 		"displayName" : "Outstations",
 		"properties" : {
-			"linkid" : {
+				"linkid" : {
 					"description" : "The outstation link ID",
 					"displayName" : "Link ID",
 					"type" : "integer",
 					"maximum" : "65519",
 					"minimum" : "1",
 					"default" : "10"
-			},
-			"address" : {
+				},
+				"address" : {
 					"description" : "The outstation TCP address or name ",
 					"displayName" : "TCP Address",
 					"type" : "string",
 					"default" : "127.0.0.1",
 					"mandatory": "true"
 				},
-			"port" : {
+				"port" : {
 					"description" : "The outstation TCP port",
 					"displayName" : "TCP Port",
 					"type" : "integer",
 					"default" : "20000",
 					"maximum" : "65000",
 					"minimum" : "1"
+				},
+				"disableTLS": {
+					"description" : "Disable TLS",
+					"displayName" : "Disable TLS",
+					"type" : "boolean",
```

Comment:
I read this that it allows you to override the default setting applied to all out stations on a per outstation basis. I think I would suggest this is a three option enumeration. "Use local default", "Enable TLS", "Disable TLS". That way if you have an outstation that you want to be sure is always configured a certain way, then you can. As you have it, if you wanted an outstation always enabled for TLS you could don't do that in a way that was imp[acted by the global setting.